linux-anvil-cuda/Dockerfile

```
@@ -85,7 +85,7 @@ RUN source /opt/conda/etc/profile.d/conda.sh && \
 RUN source /opt/conda/etc/profile.d/conda.sh && \
     conda activate && \
     conda create -n test --yes --quiet --download-only \
-        defaults::cudatoolkit=${CUDA_VER} \
+        conda-forge/label/test::cudatoolkit=${CUDA_VER} \
```

Let's get this into the `main` label before we start deploying it ( https://github.com/conda-forge/cudatoolkit-feedstock/issues/15 ) 🙂

---

Looks like those packages should be available soon. So would suggest the following change.

```suggestion
        conda-forge::cudatoolkit=${CUDA_VER} \
```

Otherwise LGTM! Thanks again for all of the work here! 😄